pkg/api/api0/errors.go

```diff
@@ -85,7 +85,7 @@ func (n ErrorCode) Message() string {
 	case ErrorCode_JSON_PARSE_ERROR:
 		return "Error parsing json response"
 	case ErrorCode_UNSUPPORTED_VERSION:
-		return "The version you are using is no longer supported"
+		return "The version you are using is no longer supported. Update Northstar to continue."
```

Comment:
nit: `The version you are using is no longer supported; update Northstar to continue`

since the other ones are also sentence fragments without a period